runtime/index.d.ts

```diff
@@ -15,9 +15,7 @@ declare module "@sapper/app" {
 	export const stores: () => unknown;
 }
 
-declare module "@sapper/server" {
-	import { Handler, Req, Res } from '@sapper/internal/manifest-server';
```

Comment:
I'm surprised these are removed. It looks like `Req` and `Res` are used a few lines later

Reply:
You're right. It's also weird it doesn't give any errors anywhere with them removed. I'll put them back.